packages/create-next-app/templates/index.ts

```diff
@@ -282,6 +282,13 @@ export const installTemplate = async ({
   const devDeps = Object.keys(packageJson.devDependencies).length;
   if (!devDeps) delete packageJson.devDependencies;
 
+  /* For pnpm v10+ x-ref: https://github.com/vercel/next.js/issues/83158 */
+  if (packageManager === "pnpm") {
+    packageJson.pnpm = {
+      onlyBuiltDependencies: ["sharp"],
```

Comment:
> There's still two other postinstall that are ignored if you use tw and eslint's next config:

what about also including the `@tailwindcss/oxide` and `unrs-resolver` here?

since they are trusted and common packages, I would suggest adding them always (unconditionally), but I can understand the approach of adding them conditionally

```suggestion
      onlyBuiltDependencies: [
        "@tailwindcss/oxide",
        "sharp",
        "unrs-resolver",
      ],
```